src/components/InsideModal/index.js

```diff
@@ -22,35 +22,45 @@ CloseBtn.defaultProps = {
   onClick: undefined,
 };
 
-export default function InsideModal({
-  show,
-  onClose,
-  appendDOM,
-  children,
-  title,
-}) {
-  const closeBtn = (<CloseBtn onClick={onClose} />);
-  if (!appendDOM) return null;
-  return (
-    <Modal
-      title={title}
-      headerClassName={styles.title}
-      className={styles.container}
-      maskClassName={styles.mask}
-      modalClassName={styles.modal}
-      contentClassName={styles.content}
-      closeBtn={closeBtn}
-      show={show}
-      appendDOM={appendDOM}>
-      {children}
-    </Modal>
-  );
+export default class InsideModal extends Component {
+  componentDidMount() {
+    this.forceUpdate();
```

Comment:
why do we call `forceUpdate` here?

Reply:
Because the `ref` modal inject node is able to get real DOM node when component did mount and force update the component.